### PR TITLE
Show the recently released panels for all users

### PR DESCRIPTION
**Pull request by jcmorrow**
Including anonymous users, and also remove their feature flags from the ExperimentalSettings section.
    
Fixes https://github.com/RecordReplay/devtools/issues/5125
Fixes https://github.com/RecordReplay/devtools/issues/5056

### CODE REVIEW

File: src/ui/components/shared/UserSettingsModal/ExperimentalSettings.tsx [hunk 1]
```diff
@@ -14,31 +14,11 @@ interface ExperimentalSetting {
...
-  {
-    label: "Elements pane",
-    description: "Inspect HTML markup and CSS styling",
-    key: "showElements",
-  },
   {
     label: "Repainting",
     description: "Repaint the DOM on demand",
     key: "enableRepaint",
   },
```
Comment (jasonLaster) [inline src/ui/components/shared/UserSettingsModal/ExperimentalSettings.tsx:21]: we can remove repainting as well

**Review by jasonLaster — COMMENTED** [review]
Will let someone else do a more thorough review, but otherwise looks good

File: test/mock/src/graphql/settings.ts [hunk 2]
```diff
@@ -9,12 +9,8 @@ export function createUserSettingsMock(): MockedResponse[] {
     defaultWorkspaceId: null,
     disableLogRocket: false,
     enableEventLink: false,
-    enableNetworkMonitor: false,
-    enableGlobalSearch: false,
     enableRepaint: false,
     enableTeams: true,
```
Comment (jasonLaster) [inline test/mock/src/graphql/settings.ts:13]:
```
    enableRepaint: false,
    enableTeams: true,
```

are these needed?

**Comment by jaril** [comment]
@ceceliacreates in case it's relevant for docs 👋

**Comment by ceceliacreates** [comment]
> @ceceliacreates in case it's relevant for docs 👋

@jaril is the MVP for tagging doc updates! Love to see it! 😍

**Pull request merged**